2023/YoctoSummit-2023.11/uefisb-yocto-part2.md

```diff
@@ -0,0 +1,227 @@
+  src="/remark-templates/3mdeb-presentation-template/images/twitter.png"
+  width="24px" style="margin-bottom:-5px; margin-left:-15px"/> @tomzy_0</a>
+- <a href="mailto:tomasz.zyjewski@3mdeb.com"><img
+  src="/remark-templates/3mdeb-presentation-template/images/email.png"
+  width="24px" style="margin-bottom:-5px; margin-left:-15px"/>
+  tomasz.zyjewski@3mdeb.com</a>
+]
+
+---
+
+# Who we are ?
+
+.center[.image-15[![](/remark-templates/3mdeb-presentation-template/images/coreboot-1024x1024.png)].image-15[![](/remark-templates/3mdeb-presentation-template/images/uefi-1024x1024.png)].image-15[![](/remark-templates/3mdeb-presentation-template/images/lvfs.png)].image-15[![](/remark-templates/3mdeb-presentation-template/images/yocto.png)]]
+.center[.image-35[![](/remark-templates/3mdeb-presentation-template/images/openpower.svg)]]
+
+- coreboot licensed service providers since 2016 and leadership participants
+- UEFI Adopters since 2018
+- Yocto Participants and Embedded Linux experts since 2019
+- Official consultants for Linux Foundation fwupd/LVFS project since 2020
+- IBM OpenPOWER Foundation members since 2020
+
+---
+
+# Recap from first part
+
+* Booting secure device
+  - Root of Trust (RoT): source that can always be trusted within a
+    cryptographic system
+  - Chain of Trust (CoT): ensuring that each subsequent element launched on the
+    device can be trusted, and thus verified by the previous element
+* Ensure that every stage of booting is verified
+* Let's focus on the process from powering up the platform to running the
+  operating system
+  - vary depending on the architecture we are dealing with
+  - in cases with UEFI Secure Boot we can talk about multistages boot chain
+    consist of BootROM, System ROM, Bootloader, OS
+
+---
+
+# Recap from first part
+
+* Explained in chapter 32 of UEFI Specification
+  - information from there should always take precedence over other documents
+* Key goal
+  - provide infrastructure for UEFI Image load time authentication
+  - UEFI Secure Boot authenticates OS Loaders and UEFI drivers
+  - Platform Owner manage platfrom's security policy and can check
+    check integrity and security of a given UEFI Image
+* UEFI Secure Boot is controlled by a set of UEFI Authenticated Variables
+  which specify UEFI Secure Boot Policy
+* UEFI Secure Boot Policy determine which images and certificates are
+  allowed and which are not allowed to be loaded
+* The subject of UEFI Secure Boot is very complex and multi-level
+  - we will cover about 1%
+  - want to focus on integrating that within Yocto Project
+* Worth to check available training courses
+  - https://p.ost2.fyi/
+
+---
+
+# Recap from first part
+
+* meta-secure-core overview:
+  - provides couple of common and platform-specific security features
+* Divided into 9 sublayers
+  - every one provide some functionality
+  - every one got its own README with more detailed description
+* Repository: https://github.com/jiazhang0/meta-secure-core
```

Comment:
let's create issue in repository to let users of that repo and author know about the fact we will use it as base for presentation

Reply:
we were not able to do this before the presentation, I am closing the thread